app/src/main/java/com/example/healthinspector/Fragments/ScanFlow/RecommendProductsFragment.java

```diff
@@ -52,8 +52,7 @@
 public class RecommendProductsFragment extends Fragment {
 
     private FragmentRecommendProductsBinding binding;
-    private static final String TAG = "RecommendProductsFragment";
-    private static final String URL_REQUEST_PRODUCTS = "https://us.openfoodfacts.org/cgi/search.pl?action=process";
+    private static final String TAG = "RecProductFragment";
```

Comment:
why did we shorten the name here? For TAG, we generally prefer to keep the full class name for easier debugging

Reply:
Oh This was because I was getting an error "Should not have a tag over 25 characters"